style.css

```diff
@@ -0,0 +1,173 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Poppins:ital,wght@0,200;1,200&family=Roboto:ital,wght@0,300;0,400;1,300;1,400&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* Navbar Section */
+.navbar {
+  width: 100%;
+  height: 60px;
+  background-color: #3c3a39;
+}
+
+.navbar-flex {
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 14px;
+  margin-right: 14px;
+}
+
+.nav-logo {
+  list-style: none;
+}
+
+.nav-logo a {
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  color: white;
+}
+
+/* Hero Section */
+.main-div {
+  background-color: #1c1a19;
+  width: 100%;
+  height: 590px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 12px;
+}
+
+.main-heading {
+  font-family: "Crete Round", serif;
+  max-width: 327px;
+  max-height: 188px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin-left: 20px;
+  margin-right: 20px;
+  margin-top: -120px;
+}
+
+.support-text {
+  font-family: "Roboto", sans-serif;
+  max-width: 327px;
+  max-height: 70px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 24px;
+  line-height: 32px;
+  color: #fff5e1;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.descriptive-text {
+  font-family: "Poppins", sans-serif;
+  max-width: 327px;
+  max-height: 176px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.accounts {
+  display: flex;
+  margin-left: 15px;
+  width: 200px;
+  height: 40px;
+}
+
+.accounts a {
+  list-style: none;
+  text-decoration: none;
+  color: #dbd8d7;
+  width: 40px;
+  height: 40px;
+}
+
+.accounts li {
+  list-style: none;
+}
+
+.main-image {
+  background-image: url("/images/main-image.svg");
+  position: absolute;
+  right: -96.24%;
+  bottom: 0.1%;
+  height: 311.7103797638283px;
+  width: 406.355061045224px;
+  left: 213px;
+  top: 232px;
+  border-radius: 0;
+  background-repeat: no-repeat;
```

Comment:
- [ ] Great job implementing the background image, but it should match the Figma design:

# Your design

![student](https://user-images.githubusercontent.com/95412943/227217387-6a8229ec-0b1f-4948-a173-f0cb0ec58357.PNG)


# Figma design

![Figma](https://user-images.githubusercontent.com/95412943/227217441-c67d8fa1-2690-4ba0-ac31-c9a92a34a970.PNG)

Kindly add the following styles so it will give you the right design
``` css
  width: 150px;
  right: 0;
  bottom: 0;
  background-position: right;